src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,112 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int GROW_FACTOR = 2;
+
+    private Node<K,V>[] nodes;
+    private int size;
+
+    @SuppressWarnings("unchecked")
+    public MyHashMap() {
+        nodes = (Node<K, V>[]) new Node[INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        size += putNode(nodes, new Node<>(key, value), false);
+        resize();
     }
 
     @Override
     public V getValue(K key) {
+        int bucket = getBucket(nodes, key);
+        Node<K, V> node = nodes[bucket];
+
+        while (node != null) {
+            if (keysEqual(node.key, key)) {
+                return node.value;
+            }
+            node = node.next;
+        }
         return null;
     }
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    @SuppressWarnings("unchecked")
```

Comment:
```suggestion

```

Reply:
Please clarify. 

I use two instances of the annotation - constructor and resize().
Both are related to creating new array, like below:
Node<K, V>[] newNodes = (Node<K, V>[]) new Node[newCapacity];

Do you mean the annotations should not be used at all or in this particular task?

Please advise. Thanks!